gists/pull-request-guidelines.md

```diff
@@ -0,0 +1,64 @@
+# Pull Request (PR) Guidelines
+
+Guidelines for creating pull requests at LaunchPad Lab.
```

Comment:
Mention here that there are separate expectations for authors, code reviewers and QA reviewers? Also a quick definition of author (person who opens the PR) would be helpful.